apis_configs/data_ingestion_job_config.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
do not add files to this folder - setup a `g3auto/` secret, so `gen3 secrets sync` automatically handles things